### PR TITLE
fixes issue #86 and issue #79

### PR DESCRIPTION
**Pull request by jameerpathan111**
Fixes issue #86 and #79

### CODE REVIEW

File: testfm/helpers.py [hunk 1]
```diff
@@ -8,14 +8,14 @@ def product():
         '-a "rpm -q satellite --queryformat=%{VERSION}" -o').read()
     project = sat_version.splitlines()[0].split(' ')[-1]
     if project.startswith('6.6'):
-        return 'sat66'
+        return ['sat66', '6.6']
```
Comment (ntkathole) [inline testfm/helpers.py]: why this change?

File: tests/conftest.py [hunk 2]
```diff
@@ -306,5 +307,17 @@ def teardown_subscribe_to_cdn_dogfood():
...
+            }))
+            for result in contacted.values():
+                logger.info(result['stdout'])
+                assert "FAIL" not in result['stdout']
+                assert result['rc'] == 0
     request.addfinalizer(teardown_subscribe_to_cdn_dogfood)
+
+
+@pytest.fixture(scope='function')
+def env_setup(monkeypatch):
```
Comment (ntkathole) [inline tests/conftest.py]: rename it to beta_env_setup

File: testfm/helpers.py (hunk 1 above)
Comment (jameerpathan111) [reply testfm/helpers.py]: I need it to do proper teardown of tests. `'version': product()[1]  # Satellite minor version`

File: tests/conftest.py (hunk 2 above)
Comment (jameerpathan111) [reply tests/conftest.py]: It will increase line length > 99

Comment (jameerpathan111) [reply tests/conftest.py]: I can add comment in there if you want.

File: testfm/helpers.py (hunk 1 above)
Comment (ntkathole) [reply testfm/helpers.py]: Make sure it won't break run_only_on decorator.

File: tests/conftest.py (hunk 2 above)
Comment (ntkathole) [reply tests/conftest.py]: you can add new line :
ex: 
```
def test_positive_beta_repositories(
                                  ansible_module, 
                                  beta_env_setup, 
                                  setup_subscribe_to_cdn_dogfood):
```

File: testfm/helpers.py (hunk 1 above)
Comment (jameerpathan111) [reply testfm/helpers.py]: It doesn't.

**Comment by ntkathole** [comment]
ok to test : tests/test_advanced.py -k "test_positive_beta_repositories, test_positive_repositories_setup"

File: tests/conftest.py (hunk 2 above)
Comment (jameerpathan111) [reply tests/conftest.py]: updated

**Comment by jameerpathan111** [comment]
ok to test : tests/test_advanced.py -k "test_positive_beta_repositories or test_positive_repositories_setup"
